docs/iivsearch.rst

```diff
@@ -203,7 +175,50 @@ algorithm for the block structure, by first choosing the number of etas then the
             s7 -> s12
 
         }
+        
+Bottom up stepwise search
+-------------------------
+
+The ``bottom_up_stepwise`` algorithm differ slightly from the ``top_down_exhaustive`` as the models are created
+in iterative steps, each adding a single ETA. The algorithm will create a model with all possible IIVs and in the first step
+it will remove all but one. This ETA will be on clearance (CL) if possible. If not, the first parameter in alphabetical order
+will have an ETA. This model is then run and its results are used to update the initial estimates of the model. In the next step,
+a candidate model is created for each remaining parameter that could have an ETA put on it. All models are run, and the best model
+is chosen for the next step, updating the initial values once more.
+
+If no better model can be found, the algorithm stops.
```

Comment:
Maybe specify what we mean by better, that we use the selection criteria specified by the user (since it could also mean using LRT or something).